### PR TITLE
Load same xclbin issue

### PR DESCRIPTION
**Pull request by chvamshi-xilinx**
Hi 
This change contains a fix to prevent downloading the same xclbin again and again for ZynqMP

### CODE REVIEW

**Comment by chvamshi-xilinx** [comment]
retest this please

**Comment by maxzhen** [comment]
retest this please

**Comment by chvamshi-xilinx** [comment]
retest this please

File: src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c [hunk 1]
```diff
@@ -317,12 +317,19 @@ int zocl_pcap_download_ioctl(struct drm_device *dev, void *data,
...
 	if (copy_from_user(&bin_obj, args->xclbin, sizeof(struct axlf)))
 		return -EFAULT;
 
 	if (memcmp(bin_obj.m_magic, "xclbin2", 8))
 		return -EINVAL;
+	
+	/* Check unique ID */
+	if (bin_obj.m_uniqueId == zdev->unique_id_last_bitstream) {
+		DRM_INFO("The XCLBIN already loaded. Don't need to reload.");
+		return ret;
```
Comment (larry9523) [inline src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c]: we don't really need a ret here, right? just return 0.

Comment (chvamshi-xilinx) [reply src/runtime_src/core/edge/drm/zocl/zocl_ioctl.c]: Fixed. Removed this as it used only in one place.

**Comment by chvamshi-xilinx** [comment]
retest this please

**Pull request merged**